fe/fe-core/src/main/java/org/apache/doris/catalog/Table.java

```diff
@@ -176,16 +188,49 @@ public boolean writeLockIfExist() {
             this.rwLock.writeLock().unlock();
             return false;
         }
+        lockWriteStart = System.currentTimeMillis();
         return true;
     }
 
+    private void checkAndLogLockDuration(String type, long lockStart, long lockEnd) {
+        long duration = lockEnd - lockStart;
+        if (duration > Config.lock_reporting_threshold_ms) {
```

Comment:
The Config lock_reporting_threshold_ms is not mutable, may be you should use a new independent and mutable config